README.md

```diff
@@ -1,2 +1,21 @@
+![Logo](Resources/logo-black-transparent.png)
 # Aquarius
-Contains the source code for Aquarius, a 2D game engine for the ECE 5010 junior software design project
+A 2D game engine developed by Colton Smith, Daniel Burke, & Liam Warren for the Junior Software Design Project (ECE 5010).
+
+## Additional Required Software
+- CMake Version 3.10 ([Download](https://cmake.org/download/))
+
+## Building Aquarius
+1. Unzip the ZIP File
+2. Navigate in command-line interface to the unzipped project folder 
+3. Make a build directory by executing the following command `mkdir build`
+4. Navigate to the build directory by executing the following command `cd build`
+5. Execute CMake to build the project by executing the following command `cmake ..`
```

Comment:
Do we want to include scripts at the root of the project (can have a .sh and a .bat so that we don't have a scripting language like python as a dependency) that does this bit for you?